src/test/java/com/google/javascript/cl2dts/dict.js

```diff
@@ -0,0 +1,13 @@
+goog.provide('dict');
+
+/**
+ * @const
+ * @dict
+ */
+dict.untyped = {};
+
+/**
+ * @dict
+ * @type {{a: Function}}
+ */
+dict.typed = {'a': function() {}};
```

Comment:
How about one with @constructor too 
```
/**
 * @constructor
 * @dict
 */
function Foo() {}
var obj1 = new Foo();
```
taken from https://developers.google.com/closure/compiler/docs/js-for-compiler?hl=en